apps/metadata-editor/src/app/records/records-list.component.html

```diff
@@ -29,6 +29,16 @@ <h1 class="text-[56px] font-title grow">{{ title }}</h1>
           >dashboard.records.publishedRecords</span
         ></a
       >
+      <gn-ui-button
+        type="light"
+        *ngIf="userCount > 0"
+        (buttonClick)="showUsers()"
+      >
+        {{ userCount }}&nbsp;
+        <span translate [translateParams]="{ count: userCount }">
+          dashboard.records.users
+        </span>
+      </gn-ui-button>
```

Comment:
New logic without new test?

Reply:
This should actually be a link instead of a button, but it was made like that initially and I am not going to invest more time in this. Also this will need to be reworked to match the new mockups anyway